src/referencePackages/src/microsoft.codeanalysis.common/4.0.1/lib/netstandard2.0/Microsoft.CodeAnalysis.cs

```diff
@@ -0,0 +1,5047 @@
+        Node = 0,
+        Token = 1,
+        Trivia = 2,
+        StructuredTrivia = 3,
+    }
+    public partial struct TypedConstant : System.IEquatable<Microsoft.CodeAnalysis.TypedConstant>
+    {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public bool IsNull { get { throw null; } }
+        public Microsoft.CodeAnalysis.TypedConstantKind Kind { get { throw null; } }
+        public Microsoft.CodeAnalysis.ITypeSymbol Type { get { throw null; } }
+        public object Value { get { throw null; } }
+        public System.Collections.Immutable.ImmutableArray<Microsoft.CodeAnalysis.TypedConstant> Values { get { throw null; } }
+        public bool Equals(Microsoft.CodeAnalysis.TypedConstant other) { throw null; }
+        public override bool Equals(object obj) { throw null; }
+        public override int GetHashCode() { throw null; }
+    }
+    public enum TypedConstantKind
+    {
+        Error = 0,
+        Primitive = 1,
+        Enum = 2,
+        Type = 3,
+        Array = 4,
+    }
+    public readonly partial struct TypeInfo : System.IEquatable<Microsoft.CodeAnalysis.TypeInfo>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public Microsoft.CodeAnalysis.NullabilityInfo ConvertedNullability { get { throw null; } }
+        public Microsoft.CodeAnalysis.ITypeSymbol ConvertedType { get { throw null; } }
+        public Microsoft.CodeAnalysis.NullabilityInfo Nullability { get { throw null; } }
+        public Microsoft.CodeAnalysis.ITypeSymbol Type { get { throw null; } }
+        public bool Equals(Microsoft.CodeAnalysis.TypeInfo other) { throw null; }
+        public override bool Equals(object obj) { throw null; }
+        public override int GetHashCode() { throw null; }
+    }
+    public enum TypeKind : byte
+    {
+        Unknown = (byte)0,
+        Array = (byte)1,
+        Class = (byte)2,
+        Delegate = (byte)3,
+        Dynamic = (byte)4,
+        Enum = (byte)5,
+        Error = (byte)6,
+        Interface = (byte)7,
+        Module = (byte)8,
+        Pointer = (byte)9,
+        Struct = (byte)10,
+        Structure = (byte)10,
+        TypeParameter = (byte)11,
+        Submission = (byte)12,
+        FunctionPointer = (byte)13,
+    }
+    public enum TypeParameterKind
+    {
+        Type = 0,
+        Method = 1,
+        Cref = 2,
+    }
+    public sealed partial class UnresolvedMetadataReference : Microsoft.CodeAnalysis.MetadataReference
+    {
+        internal UnresolvedMetadataReference() : base (default(Microsoft.CodeAnalysis.MetadataReferenceProperties)) { }
+        public override string Display { get { throw null; } }
+        public string Reference { get { throw null; } }
+    }
+    public enum VarianceKind : short
+    {
+        None = (short)0,
+        Out = (short)1,
+        In = (short)2,
+    }
+    public static partial class WellKnownDiagnosticTags
+    {
+        public const string AnalyzerException
```

Comment:
Consider pulling these into a separate file so that we can re-run the generator.  It will also help callout the manual fixes.